apps/cli/src/actions/deploy/index.ts

```diff
@@ -0,0 +1,65 @@
+import inquirer from "inquirer";
+
+import { ExerciseSettingsFile } from "../../types";
+import { fileExists, readFile, writeFile } from "../../utils";
+import { getDefaultConfigFilePath } from "../init";
+import deployExercise from "./deployExercise";
+import getExerciseFiles from "./getExerciseFiles";
+import { DeployOptions } from "./types";
+
+export async function deploy({
+  apiUrl,
+  filePath = getDefaultConfigFilePath(),
+  dryRun = false,
+}: DeployOptions) {
+  const exists = await fileExists(filePath);
+  if (!exists) {
+    console.error("Settings file not found. Please run init command before the deploy.", { filePath });
+    process.exit(1);
+  }
+
+  const configFileContent = await readFile(filePath);
+
+  let settings: ExerciseSettingsFile;
+  try {
+    settings = JSON.parse(configFileContent);
+  } catch (e) {
+    console.error("Unable to parse settings file content.", { filePath }, e);
+    process.exit(1);
+  }
```

Comment:
We might not want to do it now, but this should perhaps be something external to this command. We could consider refactoring it in another place in the future...